ckanext/xroad_integration/logic/action.py

```diff
@@ -27,7 +27,7 @@
 
 DEFAULT_TIMEOUT = 3  # seconds
 DEFAULT_DAYS_TO_FETCH = 1
-DEFAULT_LIST_ERRORS_HISTORY_IN_DAYS = 60
+DEFAULT_LIST_ERRORS_HISTORY_IN_DAYS = 90
```

Comment:
How does history in days affect anything if the errors are fetched elsewhere ?

Reply:
This is used in "xroad_error_list(context, data_dict)" for displaying last 90 days of error logs from the db under the organization view in "X-Road errors" tab, together with pagination of 20 results per page.